bin/check_tron_jobs.py

```diff
@@ -172,12 +177,32 @@ def get_relevant_action(action_runs, last_state):
     return action_runs[-1]
 
 
+def guess_realert_every(job):
+    try:
+        job_next_run = job.get('next_run', None)
+        job_runs = job.get('runs', [])
+        job_previous_run = max(
+            [run for run in job_runs if run['start_time'] is not None], key=lambda k: k['start_time'],
+        ).get('start_time')
+        # time interval in minutes
+        time_diff = (mktime(_timestamp_to_timeobj(job_next_run)) -
+                     mktime(_timestamp_to_timeobj(job_previous_run))) / 60
+        # check_tron_job runs every 5 minutes
+        realert_every = max(int(time_diff / 5), 1)
+    except Exception:
```

Comment:
Maybe we should log the exception, in case it's caused by a bug in our script? Not sure what kind of exceptions you're anticipating.

Reply:
I would rather check for the cases that you expect (e.g. next_run is None) and explicitly return -1 in those cases, so that the remaining exceptions are more meaningful.

Reply:
Yeah, that sounds better. qui++